### PR TITLE
Milestone 4: Calculations

### PR DESCRIPTION
**Pull request by SotoArmando**
Let's put aside the React work for a moment. We will now focus on the heart of the calculator, its logic. In this milestone we will implement two JavaScript modules: calculate.js and operate.js which will be responsible for all calculations. Finally, we will include those modules in the App component so we can hook them up to other components in the next milestone.

### CODE REVIEW

File: src/Logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const Big = require('big.js');
```
Comment (bellom) [inline src/Logic/operate.js:1]: Let's follow the required change committed in the package.json file -> `big-js` not `big.js`

File: src/Logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+export default function operate(numberOne, numberTwo, operation) {
```
Comment (bellom) [inline src/Logic/operate.js:4]: Let practice Dry Code -> do not repeat;
you can create a variable that will hold `new Big(numberOne)` and `new Big(numbeTwo)`.
Of which if you're using `big-js`, I don't think you need `toFixed` and `new Big`.
`new Big` can just be written as  `Big(numberOne)`

File: src/Logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  const operationssrt = ['+/-', '%', '÷', 'X', '-', '+'];
+  let x; let y; let
+    z;
+  switch (operation) {
+    case operationssrt[0]:
+      return new Big(numberOne).mul(-1).toFixed();
```
Comment (bellom) [inline src/Logic/operate.js:10]: I'm curious: `mul` and `times` are they the same thing, are they going to work in the way?
https://github.com/favio41/big-js#readme
I couldn't find `mul` in the doc 🤔

File: src/Logic/operate.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    case operationssrt[1]:
+      return new Big(numberOne).mod(new Big(numberTwo)).toFixed();
```
Comment (bellom) [inline src/Logic/operate.js:12]: case: %  -> we are not returning the reminder, we need to get the percentage

could be something like this: 
numA.times(numB).times(100)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @SotoArmando :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**1. Kindly check out my comments below;**

**2. For some reasons the linter's check is missing on your GitHub page 🤔kindly fix that** 
you should have 👇 
image


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by SotoArmando** [comment]
Alright i will close this PL main reason that Stickler CI is not working on it. So i ll try to remake it.

File: src/Logic/operate.js (hunk 3 above)
Comment (SotoArmando) [reply src/Logic/operate.js:10]: well mul was the one that pull out the what i was looking on i tried lot ways. what can i say.

File: src/Logic/operate.js (hunk 4 above)
Comment (SotoArmando) [reply src/Logic/operate.js:12]: i will do it but if is not like that you ll pay for it.

File: src/Logic/operate.js (hunk 2 above)
Comment (SotoArmando) [reply src/Logic/operate.js:4]: when i tried to print out the results it wasnt  letting me do it with Big Object. once i used -toFixed i was on the way i needed to put them out. without it is a matter on a option that i have right now. pls if you know one better you could say that. thanks for Dry code practice btw :D 👍

**Pull request closed without merging**